extensions/arc/src/common/utils.ts

```diff
@@ -218,6 +218,61 @@ export function createCredentialId(controllerId: string, resourceType: string, i
 }
 
 /**
+ * Calculates the gibibyte (GiB) conversion of a quantity that could currently be represented by a range
+ * of SI suffices (E, P, T, G, M, K, m) or their power-of-two equivalents (Ei, Pi, Ti, Gi, Mi, Ki)
```

Comment:
I know we copied this from the k8s documentation, but I'm pretty sure it's supposed to say `suffixes` instead of `suffices`